automated/linux/imagenet-resnet50/imagenet-resnet50.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-a <home-directory>]
+                    [-m <mount-directory>]
+                    [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "a:m:t:s:" o; do
+    case "$o" in
+        a) HOME_DIR="${OPTARG}";;
+        m) MNT_DIR="${OPTARG}" ;;
+        t) MNT_EXSTS="${OPTARG}" ;;
+        s) SKIP_INSTALL="${OPTARG}" ;;
+        *) usage ;;
+    esac
+done
+
+pkgs="build-essential git python3-venv python3-dev libhdf5-dev gcc-aarch64-linux-gnu"
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+create_out_dir "${OUTPUT}"
+
+rm -rf "${HOME_DIR}"/tf_venv
+rm -rf "${HOME_DIR}"/src
+rm -rf "${MNT_DIR}"/datasets
+mkdir "${HOME_DIR}"/tf_venv
+pushd "${HOME_DIR}"/tf_venv || exit
+python3 -m venv .
+source bin/activate
+if [ "${SKIP_INSTALL}" = false ]; then
+
+    python -m pip install --upgrade pip wheel
+    python -m pip install h5py
+    python -m pip install cython
+    python -m pip install google protobuf==3.20.1
+    python -m pip install --no-binary pycocotools pycocotools
+    python -m pip install absl-py pillow
+    python -m pip install --extra-index-url https://snapshots.linaro.org/ldcg/python-cache/ numpy==1.19.5
+    python -m pip install --extra-index-url https://snapshots.linaro.org/ldcg/python-cache/ matplotlib
+    python -m pip install ck
+    ck pull repo:ck-env
+    python -m pip install scikit-build
+    python -m pip install --extra-index-url https://snapshots.linaro.org/ldcg/python-cache/ tensorflow-io-gcs-filesystem==0.21.0 h5py==3.1.0
+    python -m pip install tensorflow-aarch64==2.7.0
+    mkdir "${HOME_DIR}"/src
+    pushd "${HOME_DIR}"/src || exit
+    git clone https://github.com/mlcommons/inference.git
+    pushd inference || exit
+    git checkout r1.1
+    git cherry-pick -n 215c057fc6690a47f3f66c72c076a8f73d66cb12
```

Comment:
Usage the get_test_program() for clone and checkout.
Refer the file: automated/lib/sh-test-lib
get_test_program ${TEST_GIT_URL}  \
${TEST_DIR} \
${TEST_PROG_VERSION} \
${TEST_PROGRAM}

Reply:
Thanks for the info on get_test_program. If you could double check that I've implemented it correctly I'd appreciate it.